types.go

```diff
@@ -388,6 +388,23 @@ const (
 	AddJSONPath = "/ct/v1/add-json" // Experimental addition
 )
 
+// APIEndpoint is a string that represents one of the Certificate Transparency
+// Log API endpoints.
+type APIEndpoint string
+
+// Certificate Transparency Log API endpoints - should match the URI paths
+// without the "/ct/v1/" prefix.
+const (
+	AddChainStr          APIEndpoint = "add-chain"
```

Comment:
Not sure whether these strings should be hard-coded above in lines 379-386, or used in a way like `AddChainPath = V1Prefix + AddChainStr`. The latter would be less prone to inconsistencies if the values were to change, but the way it is now is probably more readable.

Reply:
As discussed in person, I was up for this, and did it, but then it caused some type issues with the paths, and there was no simple pretty/readable to keep it that way and also give the new strings the sort of string-enum behaviour intended, so have now changed it back.  WDYT?